Backend/starter_project_mavericks/BlogApp/tests/BlogApp.UnitTests/Ratings/Create_RatingCommandHandlerTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using AutoMapper;
+using BlogApp.Application.Contracts.Persistence;
+using BlogApp.Application.Exceptions;
+using BlogApp.Application.Features.Ratings.CQRS.Commands;
+using BlogApp.Application.Features.Ratings.CQRS.Handlers;
+using BlogApp.Application.Features.Ratings.DTOs;
+using BlogApp.Application.Profiles;
+using BlogApp.Application.Responses;
+using BlogApp.UnitTests.Mocks;
+using Moq;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace BlogApp.UnitTests.Ratings;
```

Comment:
Under Rating feature you can add two folders to help separate query hanlers tests from command hanlers tests.

add folders named Query and Command...I can see that you haven't implemented query related endpoints for rating wich is okay depending what you have discussed within your team but to set a folder structure model for testing you can just add the folders.